### ISSUE TITLE
Paginate task listing

### ISSUE THREAD

**Issue by ghost** [Factual/skuld#12]
Task listing should take limit and greater-than some-id constraints.

**Comment by danielcompton** [comment]
Do you also want a next command? Maintaining the state between requests might not be desirable though.

**Comment by aphyr** [comment]
Naw, I think it's fine to drop some ops. Best-estimate instead of correctness is critical for this performing well, haha. Traversing backwards in time should be relatively stable anyway.

**Comment by danielcompton** [comment]
So each pagination request is separate with no shared state? Do we want a less than id param if we're traversing backwards?

**Comment by aphyr** [comment]
Yeah, I'd say less than and greater-than are reasonable. There's probably an extended discussion to happen around the sort-merge equi-join and how much estimation is acceptable. My guess is it's probably ideal to estimate a range based on density then return every result with an ID between a lower and upper ID bound, rather than doing a hard-limit and throwing away a bunch of IDs.
